### DISCUSSION TITLE
How to contact CKAN by mail

### DISCUSSION THREAD

**Discussion by IotaLion** [ckan/ckan#7488] [General]
Hi there,

we are a research group at OFFIS based in Oldenburg, Germany, and we need some help with our deployment of CKAN. Is there any way to contact you by mail (because of some private reasons)?

It would really help us. 
Thanks in advance.

**Comment by pdelboca** [comment]
Hello @IotaLion !

Sadly there is no way to "contact us" since we are an open community mostly based in collaborators. 

CKAN (as a project) has:
- Community Support: https://ckan.org/community
- Commercial Support: https://ckan.org/commercial (might be others as well)

If you have a specific question or a specific pain point and would like a private chat with someone (without all the hassle of going through vendors and stuff), you could also try with the folks at the OKFN: https://www.okfn.org/. 

Bests!

**Reply by IotaLion** [reply]
Hello @pdelboca 

thank you very much for the reply.
I need to exchange with my group leader first and might send a message to the OKFN then.

**Comment by thegostev** [comment]
Hi @IotaLion you can always message me alex.gostev@ckan.org, I'll provide the knowledge I have or will provide you with a relevant contact. Regarding installation, Brett is the go-to advisor who can be reached at: brett.jones@linkdigital.com.au

**Comment by knef2** [comment]
Hi,

We would like to get in touch with a data steward in terms of CKAN features. Specifically we are looking to spin up a demo instance of CKAN and trial out the features. It would be great to speak to someone who is using CKAN instance already

Best Regards 

Karen

**Reply by thegostev** [reply]
Hi, @knef2. The easiest way to get a CKAN instance is through GitHub Codespaces, here's a fresh guide from @kowh-ai: https://www.youtube.com/watch?v=KGTyX3Nx9IU . If you have questions about CKAN functionality, I'd advice to create a topic in discussions and leverage the collective mind of the community.
